checkov/terraform/checks/graph_checks/aws/AWS_private_MWAA_environment.yaml

```
@@ -0,0 +1,19 @@
+metadata:
+  id: "CKV2_AWS_67"
```

```suggestion
  id: "CKV2_AWS_65"
```
I didn't find any `CKV2_AWS_65` or do you have it somewhere in the pipeline already 😄

---

Uh oh, noted.. Will update it.. 😛

---

Done ✅